contrib/docker/release_notes/cioos_1.6.0.md

```diff
@@ -63,6 +63,16 @@ sudo chmod +x /usr/local/bin/docker-compose
 sudo docker-compose --version
 ```
 
+
+You will need to allow execution of the entrypoint files so that docker can run them on container start
+
+```
+cd ~/ckan/contrib/docker
+chmod 775 ./ckan-entrypoint.sh
+chmod 775 ./ckan-harvester-entrypoint.sh
```

Comment:
The other two aren't necessary because the files are committed with their appropriate permissions. Currently the mode is 100644. If I change it locally and git diff the file I can see:
```
old mode 100644
new mode 100755
```

I think we might as well do that.

Reply:
@sjbruce do you think this is worth amending?

Reply:
I've updated the permissions on all shell scripts in docker/contrib/ to 755 which should cover everything